spec/lib/meaning_cloud/topics_spec.rb

```diff
@@ -5,6 +5,14 @@
     MeaningCloud.configuration.key = nil
   end
 
+  it 'has a default topic types configuration' do
+    expect(MeaningCloud.configuration.topic_types).not_to be_nil
+  end
+
+  it 'has a default language configuration' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.